src/content/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-Flink-prometheus-openMetrics.mdx

```diff
@@ -0,0 +1,73 @@
+
+## Install and Configure Prometheus OpenMetrics
+
+Before getting Flink data into New Relic, first install our infrastructure agent, then expose your metrics by installing Prometheus OpenMetrics. 
+
+* Follow our [guided install](https://one.newrelic.com/nr1-core?account=3501165&state=a33ce5fc-d32b-e0b8-3941-30f13e4bc00b) to instrument your system with the infrastructure agent. Or if you prefer, you can [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) via the command line.
+* Install our [Prometheus OpenMetrics integration](/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration).
+
+After you've installed Prometheus OpenMetrics, then you need to configure the `nri-prometheus-config.yml` file. Add the following snippet anywhere to the file:
+
+```yml
+integrations:
+- name: nri-prometheus
+    config:
+    standalone: false
+    # Defaults to true. When standalone is set to `false`, `nri-prometheus` requires an infrastructure agent to send data.
+    emitters: infra-sdk
+    # When running with infrastructure agent emitters will have to include infra-sdk
+    cluster_name: <var>YOUR_CLUSTER_NAME_HERE</var>
+    # Match the name of your cluster with the name seen in New Relic. 
+    targets:
+        - description: <var>YOUR_DESCRIPTION_HERE</var>
+            urls: ["https://192.168.3.1:2379", "https://192.168.3.2:2379", "https://192.168.3.3:2379"]
+            tls_config:
+                ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+                cert_file_path: "/etc/etcd/etcd-client.crt"
+                key_file_path: "/etc/etcd/etcd-client.key"
+    verbose: false
+    # Defaults to false. This determines whether or not the integration should run in verbose mode.
+    audit: false
+    # Defaults to false and does not include verbose mode. Audit mode logs the uncompressed data sent to New Relic and can lead to a high log volume.
+    scrape_timeout: <var>YOUR_TIMEOUT_DURATION</var>
+    # Defaults to 30s. The HTTP client timeout when fetching data from endpoints.
+    scrape_duration: "5s"
+    worker_threads: 4
+    # Defaults to `4` For clusters with more than 400 endpoints, slowly increase the worker thread until scrape time falls between the desired `scrape_duration`.Increasing this value too much results in huge memory consumption if too many metrics are scraped at once.
+    insecure_skip_verify: false
+    # Defaults to false. Determins if the integration should skip TLS verification or not.
+timeout: 10s
+```
+
+## Get Flink metrics as dashboards
+
+After you've installed, integrated, and configured New Relic with Prometheus OpenMetrics, you can install the [ApacheFlink quickstart](). To find your dashboard in New Relic: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards > (select Apache Flink)**. 
```

Comment:
After you've installed the [ApacheFlink quickstart](https://newrelic.com/instant-observability/apache-flink) by configuring Prometheus OpenMetrics integration, you can view the metrics and logs on New Relic dashboard. To find your dashboard in New Relic: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards > (select Apache Flink)**.

Reply:
Minor thing. I know we sometimes do this > (select ...) format, but I think a more readable/conversational way to describe it would be like: 

Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards**, and select **Apache Flink**.